pybamm/expression_tree/parameter.py

```diff
@@ -90,6 +93,27 @@ def get_children_domains(self, children_list):
 
         return domain
 
+    def get_children_auxiliary_domains(self, children):
```

Comment:
this looks the same as the function in functions.py. Should it be in the base class (`Symbol`?)